src/main/java/org/spongepowered/api/world/Location.java

```diff
@@ -83,11 +83,11 @@
  * <p>Locations are immutable. Methods that change the properties of the
  * location create a new instance.</p>
  *
- * @param <E> The type of extent containing this location
+ * @param  The type of world containing this location
  */
-public final class Location<E extends Extent> implements DataHolder {
+public final class Location implements DataHolder {
```

Comment:
Uhhhh, why???

Reply:
Location as it's written currently can't use anything except World and maybe Chunk. Likewise, I'm moving Chunk away from a LocataionBasedDataHolder etc. and only making `ProtoWorld` implement that (since `ProtoWorld` can be recycled as a `GenerationRegion` of sorts as world gen gets fleshed out/refactored, and we'll have things extending `ProtoChunk`).